### ISSUE TITLE
Does not generate CVAR table. 

### ISSUE THREAD

**Issue by aaronbell** [daltonmaag/vttLib#31]
It appears that vttLib does not support CVAR table generation. Cascadia Code contains a CVAR of:

```
  <TSIC>
    <Version value="0x00010001"/>
    <Flags value="0"/>
    <!-- AxisCount=1 -->
    <!-- RecordCount=2 -->
    <Reserved value="0"/>
    <AxisArray index="0" value="wght"/>
    <RecordLocations index="0">
      <Axis index="0" value="-1.0"/>
    </RecordLocations>
    <RecordLocations index="1">
      <Axis index="0" value="1.0"/>
    </RecordLocations>
    <Record index="0">
      <Flags value="0"/>
      <NumCVTEntries value="3"/>
      <NameLength value="0"/>
      <CVTArray index="0" value="147"/>
      <CVTArray index="1" value="148"/>
      <CVTArray index="2" value="151"/>
      <CVTValueArray index="0" value="754"/>
      <CVTValueArray index="1" value="1004"/>
      <CVTValueArray index="2" value="765"/>
    </Record>
    <Record index="1">
      <Flags value="0"/>
      <NumCVTEntries value="3"/>
      <NameLength value="0"/>
      <CVTArray index="0" value="147"/>
      <CVTArray index="1" value="148"/>
      <CVTArray index="2" value="151"/>
      <CVTValueArray index="0" value="830"/>
      <CVTValueArray index="1" value="1085"/>
      <CVTValueArray index="2" value="843"/>
    </Record>
  </TSIC>
```
This allows the CVTs to vary across the variable axis and, in the case of Cascadia Code, I created CVTs to control the ligature positioning, and they need the CVAR table 😢 .

**Comment by aaronbell** [comment]
Question:

I read this in your `transfer.py` file:
>    - TSIC holds data for the cvar table BUT IS NOT A STRICT SUPERSET (what I found
>      with a font is that there was data in cvar that was not in TSIC -- deleting cvar
>      and recompiling TSIC in VTT compiled a smaller cvar. So theoretically both can
>      contain the same data, but as of VTT 6.33 Beta they don't have to)

Can you elaborate on what information you were finding in cvar that wasn't in TSIC?

**Comment by madig** [comment]
Fixed by #37.

> Can you elaborate on what information you were finding in cvar that wasn't in TSIC?

I found _some_ mismatch but don't have the file anymore; @robmck-ms told me that there should not be a mismatch, so it is possible I was looking at a fluke 🤷
